webui/react/src/pages/ExperimentList.tsx

```diff
@@ -162,8 +162,9 @@ const ExperimentList: React.FC = () => {
 
     // archived
     const archived = urlSearchParams.get('archived');
-    if (archived != null) {
-      filters.showArchived = (archived === '1');
+    if (archived !== null &&
+      (archived === ALL_VALUE || archived === 'unarchived' || archived === 'archived')) {
```

Comment:
nit: you could probably use the same check in  https://github.com/determined-ai/determined/pull/2333/files#diff-71021c13439e62579692aa701d15bba3fadc33b92a3a703a36fcb575a0615e67R22 (in your handleSelect)